github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -485,10 +485,10 @@ periodics:
         type: Directory
       name: audit
 - annotations:
-    k8s.v1.cni.cncf.io/networks: multus-cni-ns/sriov-passthrough-cni,multus-cni-ns/sriov-passthrough-cni
+    k8s.v1.cni.cncf.io/networks: default/sriov-passthrough-cni,default/sriov-passthrough-cni
```

Comment:
if the namespace is `default`, it would be good to update it also on the CNI plugin itself please ?

Reply:
the net-attach-def is in the default namespace - https://github.com/kubevirt/project-infra/pull/2664/commits/96b6f37ab30bdf6891d355762c09eb376824f8ca#diff-2eeb451e294b99f6885a77ec2172d26d41e04a2da0d5409456ac7a0163156081R69

Reply:
Thanks
once done we need to obsolete github/ci/prow-deploy/kustom/overlays/workloads-production/resources/multus-cni.yaml right ?

Reply:
Yes - that would come as part of decommisioning the old cluster.